packages/integration-karma/test/api/createElement/index.spec.js

```diff
@@ -105,6 +105,7 @@ describe('locker integration', () => {
                 return SecureBaseClass;
             }
         }
+        SecureBaseClass.shadow = true;
```

Comment:
This is really interesting. We will need to make some adjustments in Aura and Locker to make this work.